pkg/collectors/pool.go

```diff
@@ -127,6 +136,131 @@ func (f *PerfCollector) initPoolDescs() {
 	}
 }
 
+func isPoolInternalMDisks(poolName string, disksList rest.MDisksList) (bool, error) {
+	var internalPool bool
+	for _, disk := range disksList {
+		if poolName == disk[MdiskGroupNameKey].(string) {
+			if disk[ControllerNameKey].(string) != "" {
+				internalPool = false
+			} else {
+				internalPool = true
+			}
+		}
+	}
+	return internalPool, nil
```

Comment:
is there a reason to return an error if we always return nil?

Reply:
fixed. Removed return error